catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -32,6 +53,8 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
               height: 300,
               alt: '',
             }}
+            thumbHeight={240}
+            hasMarginBottom={false}
```

Comment:
So I've tweaked this a little since and in `catalogue/webapp/components/ImageCard/ImageCard.tsx` it's applied to a Space component to determine whether or not a margin-bottom should be applied or not.